docker-compose.reporting.yml

```diff
@@ -1,21 +1,5 @@
 version: '2.1'
 services:
-    #
-    # mender-reporting
-    #
-    mender-reporting:
-        image: mendersoftware/reporting:mender-master
-        command: server --automigrate
-        extends:
-            file: common.yml
-            service: mender-base
-        networks:
-            - mender
-        depends_on:
-            - mender-elasticsearch
-        environment:
-            REPORTING_ELASTICSEARCH_ADDRESSES: "http://mender-elasticsearch:9200"
-
```

Comment:
If we remove the main service here, we should remove the whole file

Reply:
https://github.com/mendersoftware/integration/pull/1856#discussion_r785735584

Which brings me to: it's fine to revert the whole shebang?